content/docs/start/visualization.md

```diff
@@ -0,0 +1,21 @@
+# Visualization with DVC Plots
```

Comment:
this will be part of the experiments - like start/experiments/visualization?

Reply:
It may have a deeper URL or can be like `start/visualization` if we don't plan to add too many sections to the trail.

Reply:
I'm not sure I understand this - will it be an independent trail then? Initial idea I think was to have three trails - each trail introduces DVC from a bit different starting point, but ideally touches (or gives a way to explore) all the areas. I'm not sure how we can combine this approach with an independent section. Does it makes sense to go back and discuss one more time the "start" structure?

Reply:
My original plan was to have trails consisting of "stepping stones," 100-200 words each. These stepping stones will be linked together. In total 3 trails, each having 5-7 small sections. 

But, as it looks we are going to keep the current structure more or less, visualization can be the last element in Getting Started, and in this case no need to put this in `start/experiments/visualization`. We can use `start/visualization` I believe.

Reply:
why not follow the original plan though? what is the reason to keep them on the global level?

also this way it overlaps with the metrics-params-plots section?

Reply:
> But, as it looks we are going to keep the current structure more or less

I thought we are still moving into 2-4 trails with a few sections inside. Not sure when we made the decision against it.